scripts/generate-clients.ts

```diff
@@ -0,0 +1,27 @@
+import { createFromRoot } from 'codama';
+import { rootNodeFromAnchor, AnchorIdl } from '@codama/nodes-from-anchor';
+import { renderVisitor as renderJavaScriptVisitor } from "@codama/renderers-js";
+import { SvmSpokeIdl, MulticallHandlerIdl } from "@across-protocol/contracts"
+import path from 'path';
+
+let codama = createFromRoot(rootNodeFromAnchor(SvmSpokeIdl as AnchorIdl));
+const clientsPath = path.join(__dirname, "..", "src", "svm", "clients");
+
+codama.accept(
+    renderJavaScriptVisitor(path.join(clientsPath, "SvmSpoke"))
+);
+
+codama = createFromRoot(rootNodeFromAnchor(MulticallHandlerIdl as AnchorIdl));
+codama.accept(
+    renderJavaScriptVisitor(path.join(clientsPath, "MulticallHandler"))
+);
+
+// codama = createFromRoot(rootNodeFromAnchor(MessageTransmitterIdl as AnchorIdl));
```

Comment:
we will also need MessageTransmitter client to finalize CCTP messages. It would be better if we could get IDL from upstream, but last time I checked they were not, so we ended up fetching old anchor version IDL from on-chain and converting in our contracts repo. Need to check if this has changed since the latest CCTP upgrade.